subsys/net/lib/sntp/Kconfig

```diff
@@ -19,6 +19,11 @@ config SNTP_UNCERTAINTY
 	  Enabling this option uses more resources (memory) and is not normally
 	  needed.
 
+config SNTP_GLOBAL_ADDR
+	bool "Use global IP address"
+	help
+	  Be able to set a global server IP address for the SNTP client.
```

Comment:
I am not really convinced that we want to do this. The SNTP library uses the address that is given to it so it does not need to store the address at all. This change just introduces complexity to the system as we need to administrate the server address.
Beside, if we have multiple network interfaces that uses DHCP, then each one could have their own SNTP server. So we could store the received SNTP server address into `net_if`, and then application could get it from there. Then we do not need to have these extra APIs at all.